gpii/configs/gpii.config.cloudBased.production.json

```diff
@@ -2,7 +2,7 @@
     "type": "gpii.config.cloudBased.flowManager.production",
     "options": {
         "cloudBasedUrls": {
-            "preferences": "http://preferences.gpii.net/preferences/%userToken",
+            "preferences": "@expand:kettle.resolvers.env(GPII_FLOWMANAGER_PREFERENCES_URL, https://preferences.gpii.net/preferences/%userToken)",
```

Comment:
Nothing in the architecture reads any of the members of this block "cloudBasedUrls" which highlights the fact that this is a dead, untested config. This block can be removed without any effect.